Compilation failure with vala-0.56.7

Compilation of pdfpc-4.6.0 fails with vala-0.56.7 (while it succeeded with vala-0.56.6).

**Expected Behaviour**
Compilation succeeds.

**Actual Behaviour**
```
[  2%] Generating classes/action/action_mapping.c, classes/action/link_action.c, classes/action/movie.c, classes/config_file_reader.c, classes/dbus_server.c, classes/drawings/drawing.c, classes/drawings/drawing_commands.c, classes/drawings/storage.c, classes/metadata/pdf.c, classes/metadata/xml.c, classes/options.c, classes/presentation_controller.c, classes/renderer/cache.c, classes/renderer/image.c, classes/renderer/markdown.c, classes/renderer/pdf.c, classes/rest_server.c, classes/timer.c, classes/view/behaviour/base.c, classes/view/behaviour/pdf_link.c, classes/view/markdown.c, classes/view/pdf.c, classes/view/transition_manager.c, classes/view/video.c, classes/window/fullscreen.c, classes/window/overview.c, classes/window/presentation.c, classes/window/presenter.c, classes/window/qrcode.c, interfaces/controllable.c, pdfpc.c
/tmp/portage/app-misc/pdfpc-4.6.0-r1/work/pdfpc-4.6.0/src/classes/drawings/drawing_commands.vala:57.46-57.59: error: `pdfpc.DrawingCommand' is not a supported generic type argument, use `?' to box value types
   57 |             this.drawing_commands = new List<DrawingCommand>();
      |                                              ^~~~~~~~~~~~~~    
/tmp/portage/app-misc/pdfpc-4.6.0-r1/work/pdfpc-4.6.0/src/classes/drawings/drawing_commands.vala:58.43-58.56: error: `pdfpc.DrawingCommand' is not a supported generic type argument, use `?' to box value types
   58 |             this.redo_commands = new List<DrawingCommand>();
      |                                           ^~~~~~~~~~~~~~    
/tmp/portage/app-misc/pdfpc-4.6.0-r1/work/pdfpc-4.6.0/src/classes/drawings/drawing_commands.vala:73.43-73.56: error: `pdfpc.DrawingCommand' is not a supported generic type argument, use `?' to box value types
   73 |             this.redo_commands = new List<DrawingCommand>(); // clear
      |                                           ^~~~~~~~~~~~~~             
Compilation failed: 3 error(s), 0 warning(s)
make[2]: *** [src/CMakeFiles/pdfpc.dir/build.make:108: src/classes/action/action_mapping.c] Error 1
make[1]: *** [CMakeFiles/Makefile2:169: src/CMakeFiles/pdfpc.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

**Steps to reproduce**

1. cmake ..
2. make

pdfpc version: 4.6.0
Used Distribution (GTK version, Vala version): Gentoo, gtk+-3.24.37, vala-0.56.7
Display Server (X11/Wayland): X11
HiDPI Setting (yes/no):

--- a/src/classes/drawings/drawing_commands.vala
+++ b/src/classes/drawings/drawing_commands.vala
@@ -54,8 +54,8 @@ namespace pdfpc {
         }
 
         public void clear() {
-            this.drawing_commands = new List<DrawingCommand>();
-            this.redo_commands = new List<DrawingCommand>();
+            this.drawing_commands = new List<DrawingCommand?>();
+            this.redo_commands = new List<DrawingCommand?>();
         }
 
         public void add_line(bool is_eraser,
@@ -70,7 +70,7 @@ namespace pdfpc {
 
             // After adding a new line you can no longer redo the old
             // path.
-            this.redo_commands = new List<DrawingCommand>(); // clear
+            this.redo_commands = new List<DrawingCommand?>(); // clear
 
             bool new_path = true;
             double epsilon = 1e-4; // Less than 0.1 pixel for a 1000x1000 img

